specification/maps/data-plane/Search/preview/2022-09-01-preview/geocoding.json

```
@@ -0,0 +1,961 @@
+        "operationId": "Search_PostGeocodingBatch",
+        "x-ms-examples": {
+          "A Geocoding Batch API call containing 2 Geocoding queries": {
+            "$ref": "./examples/ForwardGeocodingBatch.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/ClientId"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/Accept-Language"
+          },
+          {
+            "name": "geocodingBatchRequestBody",
+            "in": "body",
+            "description": "The list of address geocoding queries/requests to process. The list can contain a max of 100 queries and must contain at least 1 query.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/GeocodingBatchRequestBody"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/GeocodingBatchResponse"
+            }
+          },
+          "207": {
+            "description": "Partial Content. Api would return 207 when part of the batchItem fail to get a response.",
```

Note on clarity - is a lack of match (even up hierarchy) considered failure to get a response? If this is just about errors during input parsing / processing, perhaps
```suggestion
            "description": "Partial Content. One or more of the batch items could not be processed due to error. Items successfully processed but with no results are not considered partial content.",
```

---

Thanks for the suggestion! 
Yes, if no entity is matched, it won't be considered as a failure. 
Therefore, even all queries don't get any result, we will return 200 rather than 207.